modules/api/src/test/java/com/intuit/wasabi/api/ExperimentsResourceTest.java

```diff
@@ -273,24 +274,24 @@ public void postExperiment() throws Exception {
 
         when(experiments.getExperiment(newExperiment.getID())).thenReturn(experiment1);
         Response response = experimentsResource.postExperiment(newExperiment, false, AUTHHEADER);
-        assert(experiment1.equals(response.getEntity()));
+        assert (experiment1.equals(response.getEntity()));
 
         // When user(TESTUSER) doesn't have create permissions we throw an exception
         when(authorization.getUser(AUTHHEADER)).thenReturn(TESTUSER);
         doThrow(AuthenticationException.class).when(authorization)
                 .checkUserPermissions(TESTUSER, TESTAPP, Permission.CREATE);
         try {
-            experimentsResource.postExperiment(newExperiment,false, null);
+            experimentsResource.postExperiment(newExperiment, false, null);
             fail();
-        } catch (AuthenticationException ignored){
+        } catch (AuthenticationException ignored) {
         }
 
         // When user(TESTUSER) doesn't have create permissions but flags is true
         // we create a new application and add him as ADMIN_LABEL
         when(authorization.getUser(AUTHHEADER)).thenReturn(TESTUSER);
         when(experiments.getExperiment(newExperiment.getID())).thenReturn(experiment1);
         Response responseNewApp = experimentsResource.postExperiment(newExperiment, true, AUTHHEADER);
-        assert(experiment1.equals(responseNewApp.getEntity()));
+        assert (experiment1.equals(responseNewApp.getEntity()));
```

Comment:
Because I didn't change it, the autoformatter just added the spaces there... Should I go through it and replace them?